alacritty.yml

```diff
@@ -4,7 +4,8 @@
 #
 # These configuration files will be loaded in order, replacing values in files
 # loaded earlier with those loaded later in the chain. The file itself will
-# always be loaded last.
+# always be loaded last. If an import path cannot be found it will be skipped.
+# Skipped files will be logged at info level.
```

Comment:
I'm that we should mention such that something will be printed into logs, I think I'd remove this part.

Reply:
Ya I was not sure about the log info part as well. I added it to be extra explicit to the user. It will be removed.